File: app/src/main/java/com/song/deviceinfo/ui/system/SystemViewModel.java
package com.song.deviceinfo.ui.system;

import android.app.Activity;

import com.song.deviceinfo.info.SystemInfo;
import com.song.deviceinfo.ui.base.NormalViewModel;

import java.util.List;

import androidx.core.util.Pair;

/**
 * Created by chensongsong on 2020/5/27.
 */
public class SystemViewModel extends NormalViewModel {

    public List<Pair<String, String>> getSystemInfo(Activity activity) {
        return SystemInfo.getSystemInfo(activity);
    }
}

File: app/src/main/java/com/song/deviceinfo/ui/system/SystemViewModel.kt
package com.song.deviceinfo.ui.system

import android.app.Activity
import androidx.core.util.Pair
import com.song.deviceinfo.info.SystemInfo
import com.song.deviceinfo.ui.base.NormalViewModel

/**
 * Created by chensongsong on 2020/5/27.
 */
class SystemViewModel : NormalViewModel() {
    fun getSystemInfo(activity: Activity): List<Pair<String, String>> {
        return SystemInfo.getSystemInfo(activity)
    }
}